oci/spec_opts.go

```diff
@@ -1277,3 +1277,17 @@ func WithDevShmSize(kb int64) SpecOpts {
 		return ErrNoShmMount
 	}
 }
+
+// WithWindowsNetworkNamespace sets the network namespace for a Windows container.
+func WithWindowsNetworkNamespace(ns string) SpecOpts {
```

Comment:
Most Windows specific options seem to be in `spec_opts_windows.go`. I think this should be in there too.

Reply:
moved